torch/csrc/jit/pickler.cpp

```diff
@@ -748,6 +750,63 @@ OpCode Unpickler::readInstruction() {
               AT_ERROR("Unknown pickler class id");
           }
         });
+      } else if (
+          module_name == "torch._utils" &&
+          (class_name == "_rebuild_tensor_v2" ||
+           class_name == "_rebuild_qtensor")) {
+        bool quantized = class_name == "_rebuild_qtensor";
```

Comment:
ugh, I don't like that quantized tensors are somehow special case for the serialization format

Reply:
I agree, but this is how it works in eager serialization, so we are stuck with it if we want torch.save from torchscript to work with torch.load